Database/Migrations/20230403192512_Migration1.cs

```diff
@@ -56,6 +56,56 @@ protected override void Up(MigrationBuilder migrationBuilder)
                         onDelete: ReferentialAction.Cascade);
                 });
 
+            migrationBuilder.CreateTable(
+                name: "Announcements",
+                columns: table => new
+                {
+                    Id = table.Column<Guid>(type: "uniqueidentifier", nullable: false),
+                    AuthorId = table.Column<Guid>(type: "uniqueidentifier", nullable: false),
+                    PetId = table.Column<Guid>(type: "uniqueidentifier", nullable: false),
+                    Title = table.Column<string>(type: "nvarchar(max)", nullable: false),
+                    Description = table.Column<string>(type: "nvarchar(max)", nullable: false),
```

Comment:
As descriptions can be long, we could potentially change the db type here to something that stores strings of arbitrary length (`text`? IDK if there was a type like that). Without this we basically have some cap on description size that we don't know of, and that doesn't seem right.